tests/postgres_tests/test_aggregates.py

```diff
@@ -302,6 +302,23 @@ def test_array_agg_filter_slice(self):
         )
         self.assertCountEqual(qs, [[], [5]])
 
+    def test_array_agg_with_empty_filter_value(self):
+        with self.assertRaises(AssertionError) as context:
+            result = (
+                AggregateTestModel.objects.annotate(
+                    test_array_agg=ArrayAgg(
+                        "stattestmodel__int1",
+                        filter=Q(pk__in=[]),
+                        default=Value([]),
+                    )
+                )
+                .first()
+                .test_array_agg
+            )
+            self.assertEqual([], result)
```

Comment:
It's preferred to have an expected value as a second argument.
```suggestion
            self.assertSequenceEqual(result, [])
```